app/views/pageflow/editor/files/_file.json.jbuilder

```diff
@@ -5,9 +5,10 @@ json.partial!(partial: 'pageflow/files/file',
 json.call(file,
           :state,
           :rights,
-          :usage_id)
+          :usage_id,
+          :direct_upload_config)
 
-json.retryable(file.retryable?)
+json.retryable(file.respond_to?(:retryable?) ? file.retryable? : false)
```

Comment:
If `HostedFile` had a default implementation of `retryable` returning `false` this would not be necessary.